data/src/main/kotlin/com/antosik/benchproject/data/favourite/repo/FavouriteMoviesDataRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.antosik.benchproject.data.favourite.repo
+
+import com.antosik.benchproject.data.movies.common.database.dao.MovieDao
+import com.antosik.benchproject.data.popular.movies.entity.database.toDomain
+import com.antosik.benchproject.domain.favourite.repo.FavouriteMoviesRepository
+import kotlinx.coroutines.flow.map
+
+internal class FavouriteMoviesDataRepository(
+    private val dao: MovieDao,
+) : FavouriteMoviesRepository {
+
+    override fun getFavouriteMovies() =
```

Comment:
i think we can omit get prefix

Reply:
In the whole app, I have this prefix, I can change it but what about when I would like to update favorite movie it is ok to use the prefix `update`?